libde265/x86/sse-motion.cc

```diff
@@ -3527,9 +3527,9 @@ void ff_hevc_put_hevc_qpel_h_1_v_1_sse(int16_t *dst, ptrdiff_t dststride,
             r0 = _mm_srli_epi32(r0, 6);
 
             r1 = _mm_and_si128(r1,
-                    _mm_set_epi16(0, 65535, 0, 65535, 0, 65535, 0, 65535));
+                    _mm_set_epi16(-32766, 32767, -32766, 32767, -32766, 32767, -32766, 32767));
```

Comment:
@mholt I'm wondering why you have chosen these int16 values. The equivalent to `(0, 65535, 0, 65535, ...)` would be `(0, -1, 0, -1, ...)`.

Reply:
It was a best guess to preserve the range of ~65535, but if I recall correctly it also required some tinkering to prevent the exception.

Reply:
It is used as a simple bit-mask that is `AND`ed with the input pixel vector.

Do you want to change it to (0, -1) so that I can take your PR or should I just make the type fix on my side?